lib/Bio/EnsEMBL/DataCheck/Checks/CheckMartDatasetName.pm

```diff
@@ -0,0 +1,100 @@
+=head1 LICENSE
+
+Copyright [2018-2019] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the 'License');
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an 'AS IS' BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+package Bio::EnsEMBL::DataCheck::Checks::CheckMartDatasetName;
+
+use warnings;
+use strict;
+
+use Moose;
+use Test::More;
+use Bio::EnsEMBL::DataCheck::Test::DataCheck;
+use MartUtils qw(generate_dataset_name_from_db_name);
+use Bio::EnsEMBL::BioMart::Mart qw(genome_to_include);
```

Comment:
@james-monkeyshines: Yes, I did that reluctantly. My worries is that if we change the mart regex we need to change the DC too. Thinking about it, it makes no sense to force all the teams to have a checkout of the biomart and metadata repos for one test only.

Reply:
I made extra changes to load the list of species for vert from GitHub to remove dependancy on the ensembl-biomart repo. I can also add the DC to this page so that if we update the regex, we will remember to update this as well: https://www.ebi.ac.uk/seqdb/confluence/display/GTI/Change+regex+used+to+generate+mart+internal+names